CHANGELOG.md

```diff
@@ -8,6 +8,21 @@ adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased] - xxxx-xx-xx
 
+## v1.1.0 - 2020-01-17
+
+### Added
+
+- New endpoints [#13](https://github.com/sharetribe/flex-integration-sdk-js/pull/13)
+  - `integrationSdk.users.update_profile(/* ... */)`
+  - `integrationSdk.listings.update(/* ... */)`
+  - `integrationSdk.listings.approve(/* ... */)`
+  - `integrationSdk.listings.open(/* ... */)`
+  - `integrationSdk.listings.close(/* ... */)`
+  - `integrationSdk.images.upload(/* ... */)`
+  - `integrationSdk.availability_exceptions.query(/* ... */)`
+  - `integrationSdk.availability_exceptions.create(/* ... */)`
+  - `integrationSdk.availability_exceptions.delete(/* ... */)`
```

Comment:
Have we had this snake_case convention elsewhere in the Marketplace API JS SDK or in this Integration API SDK? If not, are you sure we want to introduce it here? It's not the most conventional casing in JS land.

Reply:
Oh, of course. It's just a mistake there in the changelog.